plugins/modules/azure_rm_keyvaultsecret_info.py

```diff
@@ -27,39 +29,145 @@
+
+RETURN = '''
+keyvaults:
+    description:
+        - List of secrets in Azure Key Vault.
+    returned: always
+    type: complex
+    contains:
+        sid:
+            description:
+                - Secret identifier.
+            returned: always
+            type: str
+            sample: "https://myVault.vault.azure.net/flexsecret/secret1/fd2682392a504455b79c90dd04a1bf46"
+        version:
+            description:
+                - Secret version.
+            type: str
+            returned: always
+            sample: fd2682392a504455b79c90dd04a1bf46
+        secret:
+            description: secret value.
+            type: str
+            returned: always
+            sample: mysecretvault
+        tags:
+            description:
+                - Tags of the secret.
+            returned: always
+            type: dict
+            sample: {"delete": "on-exit"}
+        type:
+            description:
+                - Content type (optional)
+            returned: always
+            type: str
+            sample: mysecrettype
+        attributes:
+            description:
+                - Secret attributes.
+            type: dict
+            contains:
+                created:
+                    description:
+                        - Creation datetime.
+                    returned: always
+                    type: str
+                    sample: "2019-04-25T07:26:49+00:00"
+                not_before:
+                    description:
+                        - Not before datetime.
+                    type: str
+                    sample: "2019-04-25T07:26:49+00:00"
+                expires:
+                    description:
+                        - Expiration datetime.
+                    type: str
+                    sample: "2019-04-25T07:26:49+00:00"
+                updated:
+                    description:
+                        - Update datetime.
+                    returned: always
+                    type: str
+                    sample: "2019-04-25T07:26:49+00:00"
+                enabled:
+                    description:
+                        - Indicate whether the secret is enabled.
+                    returned: always
+                    type: str
+                    sample: true
+                recovery_level:
+                    description:
+                        - Reflects the deletion recovery level currently in effect for secrets in the current vault.
+                        - If it contains 'Purgeable' the secret can be permanently deleted by a privileged user,
+                        - Otherwise, only the system can purge the secret, at the end of the retention interval.
+                    returned: always
+                    type: str
+                    sample: Recoverable+Purgeable
 '''
 
-from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
+from ansible.module_utils.azure_rm_common import AzureRMModuleBase
```

Comment:
```suggestion
from ansible_collections.azure.azcollection.plugins.module_utils.azure_rm_common import AzureRMModuleBase
```
This should be kept to use module_utils from this repo.

Reply:
Perfect! Thanks!